### PR TITLE
[Feature] [WRS-1296] SDK Monorepo

### PR DESCRIPTION
**Pull request by pietervp**
This PR introduces the concept of a monorepo to house several projects:

packages/sdk => this is the core of the repo, houses all the code needed to integrate with GraFx Studio
packages/actions => contains all Action type definitions and helper functions used in GraFx Studio Actions
pacakges/connectors => contains Connector Typescript definition files + examples on how to use them
packages/examples => contains examples on how to integrate with studio-sdk

The version number will be kept in sync over all packages, using the existing auto bump mechanism.

### CODE REVIEW

**Comment by brapoprod** [comment]
looks very clear and good to me tbh

**Comment by pietervp** [comment]
Only one 'hard to test' change:
* updated workflows (publish release, etc) without actually performing these actions on the repo

@brapoprod any idea how to double check these changes?

**Review by pkgacek — APPROVED** [review]

File: packages/sdk/dummy
Comment (Matthiee) [inline packages/sdk/dummy:1]: What is this?

**Review by Matthiee — APPROVED** [review]

File: packages/sdk/dummy
Comment (pietervp) [reply packages/sdk/dummy:1]: Whatever it was, it's gone now :wink:

**Review by Matthiee — APPROVED** [review]

**Review by Matthiee — APPROVED** [review]

**Comment by pietervp** [comment]
> Only one 'hard to test' change:
> 
> * updated workflows (publish release, etc) without actually performing these actions on the repo
> 
> @brapoprod any idea how to double check these changes?

We discussed offline, and found a way to do the tests within this branch. All is good

**Review by alexandraFlavia9 — DISMISSED** [review]

File: packages/sdk/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,17029 @@
+{
```
Comment (brapoprod) [inline packages/sdk/package-lock.json:1]: this file needs to be removed

**Review by brapoprod — APPROVED** [review]

**Review by alexandraFlavia9 — APPROVED** [review]

**Pull request merged**